test/requirements.txt

```diff
@@ -55,7 +55,7 @@ cycler==0.12.1
     # via matplotlib
 decorator==5.1.1
     # via ipython
-diffusers==0.26.3
```

Comment:
This version of `diffusers` is not compatible with the latest version of `huggingface_hub` (see [internal link](https://huggingface.slack.com/archives/C01G4UK3ETX/p1729259970932149))